doc/source/examples/integrated_gradients_mnist.ipynb

```diff
@@ -353,7 +353,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
checkout this notebook to its previous state as it is not part of this PR

Reply:
@gipster

Reply:
I think it's solved now